extensions/arc/package.json

```
@@ -11,19 +11,30 @@
     "Productivity"
   ],
   "license": "MIT",
+  "contributors": [
+    "zach"
+  ],
   "commands": [
     {
       "name": "search-history",
       "title": "Search History",
       "subtitle": "Arc",
       "description": "Search your browsing history.",
       "mode": "view"
+    },
+    {
+      "name": "new-window",
+      "title": "New Window",
```

Let's add a verb here to fit the naming. Maybe "Open" or "Create"?
```suggestion
      "title": "Open New Window",
```